### PR TITLE
Change Super interface

### PR DESCRIPTION
**Pull request by azu**
To follow the estree spec, change `SuperExpression` to `Super`.

- https://github.com/estree/estree/blob/master/es6.md#expressions
- https://github.com/estree/estree/pull/57

What do you think about this change?

/cc @Constellation @nzakas 

## Related issues

- https://github.com/estools/escope/issues/58
- https://github.com/twada/babel-plugin-espower/issues/2

### CODE REVIEW

File: estraverse.js [hunk 1]
```diff
@@ -269,7 +269,7 @@
         ReturnStatement: ['argument'],
         SequenceExpression: ['expressions'],
         SpreadElement: ['argument'],
-        SuperExpression: ['super'],
+        Super: ['super'],
```
Comment (nzakas) [inline estraverse.js]: Super has no child properties, so the array should be empty

Comment (azu) [inline estraverse.js]: done

**Comment by Constellation** [comment]
OK, I'll release it.
Because it includes breaking changes, I'll update the major number.

**Comment by azu** [comment]
@Constellation Thanks!

**Pull request merged**